app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -63,5 +63,48 @@ def get_subjects
         }
       end
     end
+
+    attribute(:Route)                                 { get_route }
+    attribute(:IsSalaried)                            { is_salaried? }
+    attribute(:Mod)                                   { object.description }
+    attribute(:IncludesPgce)                          { get_include_pgce }
+
+    attribute(:FullTime)                              { object.part_time? ? 3 : 1 }
+    attribute(:PartTime)                              { object.full_time? ? 3 : 1 }
+
+    def get_route
+      route_names = {
```

Comment:
I think this should be something like `#program_type_description` on `Course`

Reply:
Tried and failed heres the extract of the two methods
```def program_type_description
    if school_direct_salaried_training_programme? then " with salary"
    elsif pg_teaching_apprenticeship? then " teaching apprenticeship"
    else ""
    end
  end

  def description
    study_mode_string = (full_time_or_part_time? ? ", " : " ") +
      study_mode_description
    qualifications_description + study_mode_string + program_type_description
  end```

Reply:
hmmm in fact not too quite sure what you on about, 

what i really want it to translate 
underscores and casing and somehow throw in the brackets
school_direct_salaried_training_programme: "School Direct (salaried) training programme"